src/geo/transform.js

```diff
@@ -76,6 +76,13 @@ class Transform {
     pixelMatrix: Float64Array;
     pixelMatrixInverse: Float64Array;
 
+    // From world coordinates to, relative-to-center coordinates.
+    // Relative-to-center coordinates are centered at the current map center, and rotated by the map bearing.
+    // +x basis goes towards the right, +y basis point away from the camera, +z basis points up from the map.
+    worldToCenterDistanceMatrix: Float64Array;
+    // From Mercator coordinates to relative to center coordinates
+    mercatorCenterDistanceMatrix: Float64Array;
```

Comment:
Is that used anywhere? This seems unused from groking that PR change.

Reply:
Its useful to have for generating the debug viz in the debug-page OR for a custom layer that does the same thing as the debug view, but with shader.

Reply:
I see, we can leave it then, unless we move that debug viz in the shader which I think would be useful for authoring.